static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -19,15 +20,52 @@
                 margin-top: -$gs-baseline * 3.5;
                 padding-right: $gs-gutter / 2;
                 padding-top: $gs-baseline * 3.5;
+                width: gs-span(2) + $gs-gutter;
             }
         }
     }
 
+    .menu-group--editions > &,
     .menu-group--secondary > & {
         @include mq(desktop) {
             width: 100%;
         }
     }
+
+    .menu-group--editions .menu-group & {
+        @include mq(desktop) {
+            background-color: $guardian-brand;
+            display: block;
+            height: 54px;
+            border-radius: 50%;
+            margin-right: $gs-gutter / 2;
+            text-align: center;
+            width: 54px;
+        }
+
+        &:hover,
+        &:focus {
+            // TODO: Zef!!1
+            background-color: #00d4ff;
+            color: #ffffff;
+        }
+
+        &--edition-active {
+            @include mq(desktop) {
+                background-color: $guardian-brand-light;
+                color: $guardian-brand-dark;
+            }
+        }
+    }
+
+    .menu-group--membership-actions & {
+        @include mq(desktop) {
+            margin-left: -2px;
```

Comment:
Ha! Nice catch - this is a leftover. Move it 2px to the left, indent the item by 2px and increase it's width by 2px. Doesn't make sense.